src/components/BeamContext.tsx

```diff
@@ -86,13 +87,15 @@ export function BeamProvider({ children, ...presentationProps }: BeamProviderPro
     <BeamContext.Provider value={{ ...context }}>
       <PresentationProvider {...presentationProps}>
         <SnackbarProvider>
-          {/* OverlayProvider is required for Modals generated via React-Aria */}
-          <OverlayProvider>
-            {children}
-            {/*If the drawer is open, assume it will show modal content internally. */}
-            {modalRef.current && drawerContentStackRef.current.length === 0 && <Modal {...modalRef.current} />}
-          </OverlayProvider>
-          <SuperDrawer />
+          <AutoSaveStatusProvider>
```

Comment:
What do you think about wrapping the SnackbarProvider with AutoSaveStatusProvider? Probably not necessary, but just thinking out loud if we'd ever want access to that status/errors from within the snackbar. Probably not...

Reply:
I'm happy to do it but I don't think we gain anything from it.

_other things_ call `triggerNotice()` and they'd have access. Snackbar itself... not sure why it'd need it but it won't hurt, so sure 😄